Tools/OracleDBUpdater/OracleDBUpdater/Commands/SQLCommands/AlterCommand.cs

```diff
@@ -0,0 +1,53 @@
+﻿using ConsoleHelper;
+using System;
+
+namespace OracleDBUpdater.Commands.SQLCommands
+{
+    class AlterCommand : ICommand
+    {
+        public string Query { get; }
+        private QueryExecutor queryExecutor;
+
+        public AlterCommand(string query, QueryExecutor queryExecutor)
+        {
+            Query = query;

```

Comment:
Same constructor, so it looks like abstract class is needed that will contain all the same methods.